src/abstractdataframe/io.jl

```diff
@@ -19,15 +19,15 @@ Additionally selected MIME types support passing the following keyword arguments
 # Examples
 ```jldoctest
 julia> show(stdout, MIME("text/latex"), DataFrame(A = 1:3, B = ["x", "y", "z"]))
-\\begin{tabular}{r|cc}
-        & A & B\\\\
-        \\hline
-        & Int64 & String\\\\
-        \\hline
-        1 & 1 & x \\\\
-        2 & 2 & y \\\\
-        3 & 3 & z \\\\
-\\end{tabular}
+\begin{tabular}{r|cc}
+        & A & B\\
+        \hline
```

Comment:
this is malformatted

Reply:
how can be solved it?

Reply:
as commented on slack escape `\`, e.g. `\\begin`